crest/Assets/Crest/Crest/Scripts/Interaction/ObjectWaterInteraction.cs

```diff
@@ -20,7 +20,7 @@ public class ObjectWaterInteraction : MonoBehaviour
         [Range(0f, 1f), SerializeField]
         float _noiseAmp = 0.5f;
 
-        [Range(0f, 20f), SerializeField]
+        [Range(-1f, 1f), SerializeField]
         float _weight = 6f;
```

Comment:
This default value doesn't seem to fit in the new range?

Reply:
didnt see this feedback. good point will do.